tests/js/search/testDomain.js

```diff
@@ -118,6 +119,20 @@ function create_mock_shard_with_link_table (link_table_hash) {
     return mock_shard_file;
 }
 
+describe('Domain', function () {
+    describe('get_ekn_version', function () {
+        it('should throw an exception when datadir can\'t be found', function (done) {
```

Comment:
I don't quite understand the comment you made about this; I was talking about the sync version of `it()`. As far as I know there is no async version of `expect()`; even if you do `expect(func).toThrow(err)` then `func` is executed immediately. I removed the `done` parameter from `it` and removed line 131 and it worked fine for me.